### ISSUE TITLE
Getting Error

### ISSUE THREAD

**Issue by o2rhkzqx** [TheKVT/music-dl#4]
Please check it.
![Screenshot_2024-01-30-22-44-33-143_com android chrome](https://github.com/TheKVT/music-dl/assets/104969742/29c68955-5cc1-4c55-9171-27b7fd54403b)

---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-5-78f23526e6d4> in <cell line: 22>()
     20 
     21 
---> 22 with open('/root/.config/streamrip/config.toml', 'r') as file:
     23     data = toml.load(file)
     24 data['downloads']['folder'] = "/content/music-dl/"

FileNotFoundError: [Errno 2] No such file or directory: '/root/.config/streamrip/config.toml'

**Comment by TheKVT** [comment]
have you run upper cell called `install requirements`?

**Comment by o2rhkzqx** [comment]
Yes, only after running that I was getting errors below

![Screenshot_2024-01-30-22-59-09-674_com android chrome](https://github.com/TheKVT/music-dl/assets/104969742/8c784d13-9e75-43fe-824d-b612abc9edaf)

---------------------------------------------------------------------------
HTTPError                                 Traceback (most recent call last)
[<ipython-input-7-101cfec95ccf>](https://localhost:8080/#) in <cell line: 14>()
     12 file_url = "https://raw.githubusercontent.com/nathom/streamrip/main/rip/config.toml"
     13 save_path = "/root/.config/streamrip/config.toml"
---> 14 urllib.request.urlretrieve(file_url, save_path)
     15 clear_output()
     16 print('\033[92m Installation finished!')

6 frames
[/usr/lib/python3.10/urllib/request.py](https://localhost:8080/#) in http_error_default(self, req, fp, code, msg, hdrs)
    641 class HTTPDefaultErrorHandler(BaseHandler):
    642     def http_error_default(self, req, fp, code, msg, hdrs):
--> 643         raise HTTPError(req.full_url, code, msg, hdrs, fp)
    644 
    645 class HTTPRedirectHandler(BaseHandler):

HTTPError: HTTP Error 404: Not Found

**Comment by jdboy06** [comment]
I got the same error "HTTPError"

**Comment by raalapas** [comment]
Yeah, I am still not able to use, I don't know if this error is for everyone, or just we are having. Maybe admin can confirm.
